model_compression_toolkit/core/pytorch/data_util.py

```diff
@@ -0,0 +1,161 @@
+# Copyright 2024 Sony Semiconductor Israel, Inc. All rights reserved.
```

Comment:
If these function are only used as utils for the Hessian service then I think it's better to put it under "hessian" folder, to reduce the chance of using them for other purposes without paying attention

Reply:
There is no reason not to use this for other purposes, as there is nothing hessian-specific here. That's exactly the reason it's here and not under hessian :)